indico/migrations/versions/20210705_1635_1cec32e42f65_add_reviewed_dt_to_paper_revisions.py

```diff
@@ -0,0 +1,39 @@
+"""Add reviewed_dt to paper revisions
+
+Revision ID: 1cec32e42f65
+Revises: 420195768776
+Create Date: 2021-07-05 16:35:54.667174
+"""
+import sqlalchemy as sa
+from alembic import op
+
+from indico.core.db.sqlalchemy import UTCDateTime
+from indico.modules.events.editing.models.revisions import FinalRevisionState
+
+
+# revision identifiers, used by Alembic.
+revision = '1cec32e42f65'
+down_revision = '420195768776'
+branch_labels = None
+depends_on = None
+
+schema = 'event_editing'
+table = 'revisions'
+column = 'reviewed_dt'
+constraint = 'reviewed_dt_set_when_final_state'
```

Comment:
I'd rather keep those hardcoded below - IMHO it's more readable when those things are next to the queries/operations